var/spack/repos/builtin/packages/py-pyproj/package.py

```diff
@@ -9,19 +9,18 @@
 class PyPyproj(PythonPackage):
     """Python interface to the PROJ.4 Library."""
 
-    homepage = "http://jswhit.github.io/pyproj/"
-    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
-    git      = "https://www.github.com/jswhit/pyproj.git"
+    homepage = "https://github.com/pyproj4/pyproj"
+    url      = "https://pypi.io/packages/source/p/pyproj/pyproj-2.2.0.tar.gz"
+    git      = "https://github.com/pyproj4/pyproj.git"
 
-    # This is not a tagged release of pyproj.
-    # The changes in this "version" fix some bugs, especially with Python3 use.
-    version('1.9.5.1.1', commit='0be612cc9f972e38b50a90c946a9b353e2ab140f')
-    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+    maintainers = ['adamjstewart']
```

Comment:
Given the issues that came up here, please list `citibeth` as the maintainer.  In general... when a `maintainer` is added to a package that doesn't previously have such a designation... please consider using `git blame` to see who originally built the package.

Reply:
I don't like adding people as maintainers without their consent, but I'll add you. Let me know if you want to be added to any other packages.

Reply:
I think it's reasonable that if the large majority of work on a package was done by one person (i.e. it was created by that person and never substantially updated by anyone else); then the original package author should be given the chance to be a maintainer.